packages/recurring-payments-job/src/__tests__/recurring-payments-processor.spec.js

```diff
@@ -0,0 +1,53 @@
+import { salesApi } from '@defra-fish/connectors-lib'
+import { processRecurringPayments } from '../recurring-payments-processor.js'
+
+jest.mock('@defra-fish/connectors-lib', () => ({
+  salesApi: { getDueRecurringPayments: jest.fn() }
+}))
+
+describe('recurring-payments-processor', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('console log displays "Recurring Payments job disabled" when env is false', async () => {
+    process.env.RUN_RECURRING_PAYMENTS = 'false'
+    const consoleLogSpy = jest.spyOn(console, 'log').mockImplementation(jest.fn())
+
+    await processRecurringPayments()
+
+    expect(consoleLogSpy).toHaveBeenCalledWith('Recurring Payments job disabled')
+    consoleLogSpy.mockRestore()
```

Comment:
Are all these `mockRestore()`s necessary? I don't think we do it elsewhere and it seems like we should be clearing mocks before running other tests anyway.